core/app/models/mno_enterprise/tenant_config.rb

```diff
@@ -121,6 +113,39 @@ def self.build_object(schema)
       end
     end
 
+    # Flag field as readonly in the config schema
+    # A field is readonly if the `<field>_readonly` key is `true` in the `frontend_config`
+    def self.update_readonly_fields(schema, frontend_config)
+      frontend_config.each do |current_key, value|
+        if current_key =~ /_readonly$/
+          # Flag field as RO
+          schema['properties'][current_key.gsub('_readonly', '')].merge!('readonly' => true)
```

Comment:
why not ?
`schema['properties'][current_key.gsub('_readonly', '')]['readonly'] = true`